.github/workflows/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+name: deployment
+on:
+  push:
+    branches:
+      - "trunk"
+env:
+  COMMIT_SHA: ${{ github.event.head.sha }}
+jobs:
+  build-and-push-frontend-image:
+    runs-on: ubuntu-latest
```

Comment:
Use specific version for stability

Reply:
I agree in general, but for this project idc